### DISCUSSION TITLE
[FR]Preload coming songs in playlists

### DISCUSSION THREAD

**Discussion by proItheus** [gokadzev/Musify#160] [Ideas]
My internet connection is of relatively low speed, so having to wait several seconds between two songs in a playlist. It will be nice if some kinds of preloading mechanics could be added. For example, cache next song in playlist when playing, or maybe adding an option to adjust when and how many songs to preload?
Sorry for my bad English:blush:

**Comment by bestyys** [comment]
Is Musify supporting download music? Then I do not need to download from other app or site for offline use.

**Comment by elmataviejas5469-a11y** [comment]
Add preload coming songs pls?? Or make it an option,just please,im desperate for smooth transitions
